BUILD

```diff
@@ -4,6 +4,8 @@ load("@com_github_atlassian_bazel_tools//multirun:def.bzl", "multirun")
 load("@bazel_gazelle//:def.bzl", "gazelle")
 load("@io_bazel_rules_go//go:def.bzl", "TOOLS_NOGO", "nogo")
 
+# gazelle:proto disable_global
```

Comment:
To keep these in the same place, could you move this to L173?

Reply:
sure thing!